Turf.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "Turf"
-  s.version = "2.0.0-alpha.3"
+  s.version = "2.0.0-alpha.4"
```

Comment:
Let’s make it v2.0.0-beta.1, for consistency with MapboxDirections and MapboxCoreNavigation, which have long been in beta.